cmd/minishift/cmd/start_preflight.go

```diff
@@ -44,6 +44,7 @@ import (
 
 const (
 	StorageDisk                  = "/mnt/?da1"
+	StorageDiskForGeneric        = "/"
```

Comment:
For the generic driver we can not assume that users would create machines with one `/` partition.  So I suggest we make it not applicable for generic driver. 

```
$ minishift status
Minishift:  Running
Profile:    minishift
OpenShift:  Running (openshift v3.9.0+b8265d6-31)
DiskUsage:  Not applicable (For generic VM driver)
CacheUsage: 4.33 GB (used by oc binary, ISO or cached images)
```
cc @minishift/minishift-dev

Reply:
I do not  think it will be using multiple `/` partitions.. as now we don't create our own VM but use an already created VM, the machines contents are most probably stored in `/var` or `/home/<user>` which is also generally inside `/` as for VM's on the cloud people don't make separate partitions.

For now i think this is OK

Reply:
@anjannath @gbraad In that case we should atleast mention we are showing the disk usage of `/` as it will keep things explicit for the user to understand. @praveenkumar WDYT?